mapslicehelp/mapslicehelp.go

```diff
@@ -0,0 +1,109 @@
+package mapslicehelp
+
+import (
+	"slices"
+
+	"github.com/tobshub/go-sortedmap"
```

Comment:
Waarom sortedmap én orderedmap?

Reply:
sortedmap houdt de map op volgorde van values (met een sort function).
orderedmap houdt de map op volgorde van inserten. (combinatie list en map)

voor beide hebben we een use case